packages/truffle/test/scenarios/commandRunner.js

```diff
@@ -54,7 +54,13 @@ module.exports = {
     displayHost
   } = {}) {
     const cmdLine = `${this.getExecString()} ${executableCommand} ${executableArgs}`;
-    const readyPrompt = `truffle(${displayHost})>`;
+    let readyPrompt;
+
+    if (executableCommand === "debug") {
+      readyPrompt = `debug(${displayHost})>`;
+    } else {
+      readyPrompt = `truffle(${displayHost})>`;
+    }
```

Comment:
Thoughts on writing this as 
```    
const readyPrompt = executableCommand === "debug" ? `debug(${displayHost})>` : `truffle(${displayHost})>`;
```

Reply:
I like your idea of removing `if/else` and using a ternary instead of that, @lsqproduction. I will change this. Thanks!